lib/exe/css.php

```diff
@@ -34,98 +35,125 @@ function css_out(){
 
     if ($INPUT->str('s') == 'feed') {
         $mediatypes = array('feed');
-        $type = 'feed';
     } else {
-        $mediatypes = array('screen', 'all', 'print');
-        $type = '';
+        $mediatypes = array('screen', 'all', 'print', 'handheld');
```

Comment:
I wrote that bug report in 2010 and I mentioned "handheld" only as an example as there are many more in CSS2.1. Nothing uses the "handheld" media type as media queries are used for mobile devices nowadays. But most of the others are not used either and are [deprecated as the current list is much shorter](https://drafts.csswg.org/mediaqueries-4/#media-types): `all`, `print`, `screen` and `speech`. Although that spec also says:

> It is expected that all of the media types will also be deprecated in time, as appropriate media features are defined which capture their important differences.

So, I'd suggest to not add `handheld` but `speech` instead for now.